VinylRecordsRepository.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VinylRecordsManager.Models;
+using VinylRecordsManager.Data;
+using VinylRecordsManager.View;
+using System.Runtime.Remoting.Messaging;
+
+namespace VinylRecordsManager.Data
+{
+    public class VinylRecordsRepository
+    {
+        List<VinylRecords> masterList = new List<VinylRecords>();
+        VinylRecords vinyl = new VinylRecords();
+        VinylRecordsView userEnteredId = new VinylRecordsView();
+        //List here where everything needs to save.
+        public void Create(VinylRecords record)
+        {
+            masterList.Add(record);
+        }
+
+        public void ReadAll()
+        {
+            Console.WriteLine(string.Join<VinylRecords>("\n", masterList));
+        }
+
+        public List<VinylRecords> ReadById()
+        {
+            while (true)
+            {
+                int id = userEnteredId.SearchVinylRecords();
+                List<VinylRecords> individualList = new List<VinylRecords>();
+                var x = masterList.Any(C => C.Id == id);

```

Comment:
Since we haven't learnt about LINQ yet, the way to do the find here is to do a foreach loop to go over the masterList and compare each object's id with the id passed into the parameter. If it matches then return the object. Once the loop finishes, that means the id was not found so return null.